.github/CHANGELOG.md

```diff
@@ -2,6 +2,42 @@
 
 <h3>New features since last release</h3>
 
+* "compact" decompositions as described in <https://arxiv.org/abs/2104.07561>,
+ (``rectangular_compact`` and ``triangular_compact``) are now available in
+ ``decompositions`` and as options in ``ops.Interferometer``. This decomposition
+ alllows for lower depth photonic circuits in physical devices by applying two 
+ independent phase shifts in parallel inside each Mach-Zehnder interferometer.
+ ``rectangular_compact`` reduces the layers of phase shifters from 2N+1 to N+2 
+ for an N mode interferometer when compared to e.g. ``rectangular_MZ``.
```

Comment:
```suggestion
 (``rectangular_compact`` and ``triangular_compact``) are now available in
 the ``sf.decompositions`` module, and as options in the ``Interferometer`` operation.
 [(#584)](https://github.com/XanaduAI/strawberryfields/pull/584)
 
 This decomposition allows for lower depth photonic circuits in physical devices by applying two 
 independent phase shifts in parallel inside each Mach-Zehnder interferometer.
 ``rectangular_compact`` reduces the layers of phase shifters from 2N+1 to N+2 
 for an N mode interferometer when compared to e.g. ``rectangular_MZ``.
```

Moving the PR link higher up, to match the convention on previous changelog entries